tests/lean/ppMotives.lean.expected.out

```diff
@@ -11,7 +11,7 @@ protected def Nat.add : Nat → Nat → Nat :=
 fun x x_1 =>
   Nat.brecOn (motive := fun x => Nat → Nat) x_1
     (fun x f x_2 =>
-      (match x_2, x : (x : Nat) → (x : Nat) → Nat.below (motive := fun x => Nat → Nat) x → Nat with 
+      (match x_2, x : (x x : Nat) → Nat.below (motive := fun x => Nat → Nat) x → Nat with 
```

Comment:
The old output is wrong and the new one is right. (Note: I did not figure out why it used to print that way).